src/CodeEditor.tsx

```diff
@@ -13,6 +13,44 @@ export interface ICodeEditorProps {
   onChange?: (code: string) => any;
 }
 
+const defaultSupport = {
+  css: 'CSS',
+  javascript: 'JavaScript',
+  markup: 'Markup',
+  clike: 'C-like',
+};
+
+const optionalSupport = {
+  // cpp: 'C++',
+  // csharp: 'C#',
+  // go: 'Go',
+  // html: 'HTML',
+  // http: 'HTTP',
+  // java: 'Java',
+  // json: 'JSON',
+  // jsx: 'JSX',
+  // markdown: 'Markdown',
+  // objectivec: 'Objective-C',
+  // perl: 'Perl',
+  // php: 'PHP',
+  // python: 'Python',
+  // ruby: 'Ruby',
+  // scala: 'Scala',
+  // bash: 'Bash',
+  // swift: 'Swift',
+  // yaml: 'YAML',
+};
+
+/**
+ * List of supported languages: https://prismjs.com/#languages-list
+ */
+export const supportedLanguages = {
```

Comment:
Any point of exporting this info? I mean, we import all available languages at the moment, therefore this object contains incorrect information.

Reply:
I will use that list in `slate-editor` to build a dropdown (no point duplicating this in slate-editor repo)

Reply:
What I mean is that the list of supported languages is incorrect. We should export everything that's available. If you need that only for Slate, define the list in Slate.

Reply:
What do you mean that it's incorrect? It is exactly what the CodeEditor will support right now. It fails on any language than 
```
  css: 'CSS',
  javascript: 'JavaScript',
  markup: 'Markup',
  clike: 'C-like',
```

Reply:
I don't see why Slate would define what is supported. It's not Slate that defines that but the CodeEditor. What if Slate 'tells' that "hungarian" is a supported language? But CodeEditor clearly doesn't support that :)

Reply:
Ah yeah, thought we imported everything. It's ok then, let's leave it as is.